README.md

```diff
@@ -41,7 +41,7 @@
         --tasks $TASK \
         --num_fewshot "2,3,3,3" \
         --device "cuda" \
-        --output_path "result.json"
+        --output_path "result/result.json"
```

Comment:
Maybe should we remove this change since it's not relevant?

Reply:
Thanks for the review.

There is no problem to remove this from the PR as it is not related to this PR.

However, I was confused because I got an error when I executed it according to "--output_path "result.json"" in the README.

I think this tool will be easier to use if it is corrected at some point.

Reply:
@webbigdata-jp could you share the error? I believe the error you encountered was solved by [this pr](https://github.com/Stability-AI/lm-evaluation-harness/pull/79).

Reply:
Oh, I see.
I am missing that PR.
It is lgtm, thank you for your response.

Reply:
@webbigdata-jp I will merge after you fix this. Thanks!

Reply:
@mkshing 
Yes, I have corrected it.
Thank you.